File: app/src/main/java/com/dm/bomber/BuildVars.java
package com.dm.bomber;

public class BuildVars {
    public static final String DATABASE_URL = "https://androidsmsbomber-default-rtdb.europe-west1.firebasedatabase.app";

    public static final String TELEGRAM_URL = "https://t.me/androidsmsbomber";
    public static final String SOURCECODE_URL = "https://github.com/dmitrijkotov634/android-bomber/";
    public static final String DONATE_URL = "https://smsbomber.page.link/donate";

    public static final String[] COUNTRY_CODES = {"7", "380", "375", "77", ""};

    public static final int[] COUNTRY_FLAGS = {
            R.drawable.ic_ru,
            R.drawable.ic_uk,
            R.drawable.ic_by,
            R.drawable.ic_kz,
            R.drawable.ic_all};

    public static final int[] MAX_PHONE_LENGTH = {10, 9, 9, 9, 0};

    public static final int SCHEDULED_ATTACKS_LIMIT = 4;
    public static final int MAX_REPEATS_COUNT = 10;
    public static final int REPEATS_MAX_LENGTH = String.valueOf(MAX_REPEATS_COUNT).length();
}

File: app/src/main/java/com/dm/bomber/BuildVars.kt
package com.dm.bomber

object BuildVars {
    const val TELEGRAM_URL = "https://t.me/androidsmsbomber"
    const val SOURCECODE_URL = "https://github.com/dmitrijkotov634/android-bomber/"
    const val DONATE_URL = "https://smsbomber.page.link/donate"

    const val DATA_SOURCE = "https://gitlab.com/retrocat/bomber-static/-/raw/main/static75.json?ref_type=heads"

    val COUNTRY_CODES = arrayOf("7", "380", "375", "77", "")

    val COUNTRY_FLAGS = intArrayOf(
        R.drawable.ic_ru,
        R.drawable.ic_uk,
        R.drawable.ic_by,
        R.drawable.ic_kz,
        R.drawable.ic_all
    )

    const val PHONE_ANY_LENGTH = 0

    val MAX_PHONE_LENGTH = intArrayOf(10, 9, 9, 9, PHONE_ANY_LENGTH)

    const val SCHEDULED_ATTACKS_LIMIT = 12

    const val MAX_REPEATS_COUNT = 10
    const val REPEATS_MAX_LENGTH = MAX_REPEATS_COUNT.toString().length

    enum class AttackSpeed(val chunkSize: Int) {
        SLOW(1),
        DEFAULT(8),
        FAST(17)
    }
}
